src/invoker.ts

```diff
@@ -259,7 +259,7 @@ export class ErrorHandler {
     });
 
     process.on('exit', code => {
-      logAndSendError(new Error(`Process exited with code ${code}`), latestRes);
+      console.log(`Process exited with code ${code}`);
```

Comment:
The process could exit from the user's function. I think users would lose error logs in this case.

I think we should only only do this for `SIGINT`.

Reply:
Updated `exit` to `SIGINT`.